src/lib-ssl-iostream/iostream-openssl-params.c

```diff
@@ -13,10 +13,19 @@ generate_dh_parameters(int bitsize, buffer_t *output, const char **error_r)
 {
         DH *dh;
 	unsigned char *p;
-	int len, len2;
+	int len, len2, success;
 
+#if OPENSSL_VERSION_NUMBER >= 0x00908000L
+	success = DH_generate_parameters_ex(dh, bitsize, DH_GENERATOR, NULL);
```

Comment:
iostream-openssl-params.c: In function 'generate_dh_parameters':
iostream-openssl-params.c:19:10: error: 'dh' is used uninitialized in this function [-Werror=uninitialized]
  success = DH_generate_parameters_ex(dh, bitsize, DH_GENERATOR, NULL);
          ^